src/main/java/graphql/normalized/ExecutableNormalizedOperationFactory.java

```diff
@@ -155,22 +187,47 @@ public int getMaxChildrenDepth() {
      * @param document              the {@link Document} holding the operation text
      * @param operationName         the operation name to use
      * @param coercedVariableValues the coerced variables to use
-     *
      * @return a runtime representation of the graphql operation.
      */
     public static ExecutableNormalizedOperation createExecutableNormalizedOperation(
             GraphQLSchema graphQLSchema,
             Document document,
             String operationName,
             CoercedVariables coercedVariableValues
+    ) {
+        return createExecutableNormalizedOperation(
+                graphQLSchema,
+                document,
+                operationName,
+                coercedVariableValues,
+                Options.defaultOptions());
+    }
+
+    /**
+     * This will create a runtime representation of the graphql operation that would be executed
+     * in a runtime sense.
+     *
+     * @param graphQLSchema         the schema to be used
+     * @param document              the {@link Document} holding the operation text
+     * @param operationName         the operation name to use
+     * @param coercedVariableValues the coerced variables to use
+     * @param options               the {@link Options} to use for parsing
+     * @return a runtime representation of the graphql operation.
+     */
+    public static ExecutableNormalizedOperation createExecutableNormalizedOperation(
```

Comment:
I created a version of `createExecutableNormalizedOperation` that takes `Option`